ErrorHandling.js

```diff
@@ -0,0 +1,98 @@
+class CarDamageException extends Error {}
+class InsuranceClaimException extends Error {}
+class InjuryException extends Error {}
+class AirbagDeployedException extends Error {}
+class GasLeakException extends Error {}
+
+try {
```

Comment:
why do you have so many try-catch-finally block, these are throwing all the errors. try to make it more like real-life where we may get a random exception and in the function we have try-catch-finally where we check the type of exception and handle it there.

Reply:
done